contracts/truefi2/TrueRateAdjuster.sol

```diff
@@ -87,13 +88,10 @@ contract TrueRateAdjuster is ITrueRateAdjuster, UpgradeableClaimable {
     /// @dev array of pools used to calculate TrueFi TVL
     ITrueFiPool2[] public tvlPools;
 
-    // ======= STORAGE DECLARATION END ============
-
-    /// @dev emit `pool` when adding to TVL
-    event PoolAddedToTVL(ITrueFiPool2 pool);
+    /// @dev used for TVL calculations
+    IPoolFactory public factory;
```

Comment:
As we have more than one contract which serves as a factory, I'd be in favour of renaming this to `poolFactory`.